### DISCUSSION TITLE
Delete Header

### DISCUSSION THREAD

**Discussion by bizkeliza** [globaleaks/globaleaks-whistleblowing-software#3786] [Q&A]
Hello. 
Is it possible to remove the message that appears in the header from the home page? 
<img width="898" alt="globaleaks" src="https://github.com/globaleaks/GlobaLeaks/assets/91684901/ba837e47-7555-4724-b2c3-72e21539b84a">

Thank you and best regards

**Comment by ATSandrearicci** [comment]
In Settings->Advanced check "Disable the privacy panel"
